### PR TITLE
feat: added gforms integration

### LINKED ISSUE #9: Write TypeScript scripts interacting with Google Forms
Write TypeScript (Bun) scripts interacting with Google Forms for the [Windmill Hub](https://hub.windmill.dev/).
The integration requires 6 scripts. You can find all the script descriptions in `src/integrations.yaml`.

Full instructions on how to do this can be found in the `README.md` file.

Get started by running `bun run setup gforms`.

### PR DESCRIPTION
**Pull request by ezhil56x**
Fixes #9 
/claim #9

### CODE REVIEW

File: integrations/gforms/Create_Form/script.bun.test.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	console.log(`TEST: Will test Create Form with arguments: ${title}`)
+
+	// calling main
+	console.log('TEST: Running main function')
+	const response = await main(resource, title)
+
+	// assertions here
+	expect(response).toBe(`Form ${title} created`)
+	// test the response of the main function as well as the side effects of the action directly on the service
+	const forms = google.forms({
```
Comment (HugoCasa) [inline integrations/gforms/Create_Form/script.bun.test.ts:18]: that's not checking anything

File: integrations/gforms/Create_Text_Question/script.bun.test.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	console.log(`TEST: Will test Create Text Question with arguments: ${formId}`)
+
+	// calling main
+	console.log('TEST: Running main function')
+	const response = await main(resource, formId)
+
+	// assertions here
+	expect(response).toBe(`Text Question created`)
+	// test the response of the main function as well as the side effects of the action directly on the service
+	const forms = google.forms({
```
Comment (HugoCasa) [inline integrations/gforms/Create_Text_Question/script.bun.test.ts:18]: that's not checking anything

File: integrations/gforms/Create_Text_Question/script.bun.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+		auth: resource.token
+	})
+
+	// create text question
+	const textQuestion = {
+		requests: [
+			{
+				createItem: {
+					location: {
+						index: 0
```
Comment (HugoCasa) [inline integrations/gforms/Create_Text_Question/script.bun.ts:20]: should take the index as an argument with default to 0

File: integrations/gforms/Create_Text_Question/script.bun.ts [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+					},
+					item: {
+						questionItem: {
+							question: {
+								textQuestion: {
+									paragraph: false
```
Comment (HugoCasa) [inline integrations/gforms/Create_Text_Question/script.bun.ts:26]: make it an argument as well

**Review by HugoCasa — COMMENTED** [review]
- return the created/updated object as much as possible
- actually check on the service if the action was executed

**Comment by ezhil56x** [comment]
@HugoCasa 

All test cases have passed. Can you review once?

https://github.com/windmill-labs/windmill-community-integrations/assets/103899034/dd3b26b5-b2bd-4b25-95c2-4290418acdfb

**Pull request merged**